app/queries/update_target_mutator.rb

```diff
@@ -121,7 +121,7 @@ def course
   end
 
   def level
-    @level ||= target.level
+    @level ||= target_group.level
```

Comment:
I looked into the `target_group` method, and again, it is school-scoped, when it should be course-scoped, since the target group to which a target can be moved should be in the same course.

Changing the scope in that method would also mean editing the `target_and_target_group_in_same_course` validation method to be something like `target_group_must_exist`.